weblate/templates/translate.html

```diff
@@ -208,7 +208,11 @@ <h4 class="panel-title">
                       <ul class="dropdown-menu">
                         {% if user_can_delete %}
                           <li>
-                            <a href="" class="link-post" data-href="{% url "delete-unit" unit_id=unit.pk %}">{% trans "Delete string" %}</a>
+                            <!-- use this_unit_url as the next unit will become the unit at current offset after deletion -->
```

Comment:
Are you sure this is the case? The search result should be cached in the sessions, so the offset IMHO should not change (but I didn't verify that).

Reply:
Okay, looking at the code I see that the cache is discarded upon deletion.